deploy/manifests/base/storetheindex/config

```diff
@@ -0,0 +1,61 @@
+{
+  "Identity": {
+    "PeerID": "",
+    "PrivKey": ""
+  },
+  "Addresses": {
+    "Admin": "/ip4/0.0.0.0/tcp/3002",
```

Comment:
Do we want to expose this externally, or keep it on localhost and require exec into pod?

Reply:
Two things here: 
1. binding to `0.0.0.0` does not mean it is exposed externally. This is only accessible within k8s.
2.This is how it's configured in prod.